src/bootstrap-fork.js

```diff
@@ -173,12 +173,27 @@ function terminateWhenParentTerminates() {
 }
 
 function configureCrashReporter() {
-	const crashReporterOptionsRaw = process.env['CRASH_REPORTER_START_OPTIONS'];
-	if (typeof crashReporterOptionsRaw === 'string') {
+	const crashReporterStartOptionsRaw = process.env['CRASH_REPORTER_START_OPTIONS'];
+	const crashReporterExtraParametersRaw = process.env['CRASH_REPORTER_EXTRA_PARAMETERS'];
+	const crashReporter = process['crashReporter'];
+	if (typeof crashReporterStartOptionsRaw === 'string') {
 		try {
-			const crashReporterOptions = JSON.parse(crashReporterOptionsRaw);
-			if (crashReporterOptions) {
-				process['crashReporter'].start(crashReporterOptions);
+			const crashReporterStartOptions = JSON.parse(crashReporterStartOptionsRaw);
+			if (crashReporterStartOptions) {
+				crashReporter.start(crashReporterStartOptions);
```

Comment:
I am confused, why is this needed? Isn't it enabled for all processes with Electron 9?